vuepress/docs/v7.1/docs/create/blueprint-features.md

```diff
@@ -1,6 +1,9 @@
+---
+sidebarDepth: 2
+---
 # Entando Blueprint Features
 
-The Entando Blueprint makes it easier and faster to customize your application by generating controllers, repositories, services, and micro frontends for your entity. It means generating those project files in minutes, simply by answering questions that define the parameters of your application. 
+The Entando Blueprint is an easier and faster way to customize your application by generating controllers, repositories, services, and micro frontends for your entity. The project files are generated in minutes, by entering data that define the parameters of your application. 
```

Comment:
what is the rule for the comma here? comma if following clause isn't necessary to complete the sentence/message and no comma if it is? (i agree with the comma, just confirming)

Reply:
I think a comma is needed if what comes after is not required for understanding the first clause. If it's just additional information, a comma needed.